sdk/lib/crt/misc/dbgrpt.cpp

```diff
@@ -325,11 +325,7 @@ int __cdecl _CrtDbgReport(int reportType, const char *filename, int linenumber,
 
     if (format)
     {
-        va_list arglist;
-        va_start(arglist, format);
-        int len = _vsnprintf(szFormatted, DBGRPT_MAX_BUFFER_SIZE - 2 - sizeof(DBGRPT_ASSERT_PREFIX_MESSAGE), format, arglist);
-        va_end(arglist);
-
+        int len = _vsnprintf(szFormatted, DBGRPT_MAX_BUFFER_SIZE - 2 - sizeof(DBGRPT_ASSERT_PREFIX_MESSAGE), format, argptr);
```

Comment:
```suggestion
        int len = _vsnprintf(szFormatted, DBGRPT_MAX_BUFFER_SIZE - (_countof(DBGRPT_ASSERT_PREFIX_MESSAGE) - 1), format, argptr);
```

Reply:
yeah I guess this is a bug in the original implementation.

Reply:
If this needs to be fixed, it will be in a separate PR.

Reply:
Okay.